x-pack/plugins/security_solution/common/endpoint/service/authz/authz.ts

```diff
@@ -50,181 +35,58 @@ export function hasKibanaPrivilege(
  * @param licenseService
  * @param fleetAuthz
  * @param userRoles
- * @param isEndpointRbacEnabled
- * @param permissions
- * @param hasHostIsolationExceptionsItems if set to `true`, then Host Isolation Exceptions related authz properties
- * may be adjusted to account for a license downgrade scenario
  */
-
-// eslint-disable-next-line complexity
 export const calculateEndpointAuthz = (
   licenseService: LicenseService,
   fleetAuthz: FleetAuthz,
-  userRoles: MaybeImmutable<string[]> = [],
-  isEndpointRbacEnabled: boolean = false,
-  hasHostIsolationExceptionsItems: boolean = false
```

Comment:
So where do we calculate if someone can see/have access to Host Isolation Exceptions if we don't send in this boolean `hasHostIsolationExceptionsItems`?

Reply:
It is explained in the comment above